core/src/main/resources/hudson/cli/Messages.properties

```diff
@@ -84,6 +84,7 @@ ReloadConfigurationCommand.ShortDescription=Discard all the loaded data in memor
 ConnectNodeCommand.ShortDescription=Reconnect to a node(s)
 DisconnectNodeCommand.ShortDescription=Disconnects from a node.
 QuietDownCommand.ShortDescription=Quiet down Jenkins, in preparation for a restart. Don’t start any builds.
+SafeRestartCommand.ShortDescription=Safe Restart Jenkins. Don't start any builds and try to  existing.
```

Comment:
Incomplete?

Reply:
also the single quote needs escaping by repeating the ' or changing it to be an apostrophe as it in the line above ’

Reply:
Good catch, thanks!